tool/update-staging.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -e
+# Find directory containing this script.
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
I usually use the following env variables:
```
SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
PROJECT_DIR="$( cd ${SCRIPT_DIR}/.. && pwd )"
```

Reply:
I'll steal those :)